src/@utils/subgraph.ts

```diff
@@ -200,20 +201,40 @@ const OpcFeesQuery = gql`
   }
 `
 
+async function verifyUrl(url: string): Promise<number> {
+  try {
+    const response = await axios.get(url)
+    return response.status
+  } catch (error) {
+    if (!error.response) {
+      LoggerInstance.error('Subgraph network error')
+    } else {
+      LoggerInstance.error('Subgraph network error', error.message)
+    }
+  }
+}
+
 export function getSubgraphUri(chainId: number): string {
-  const config = getOceanConfig(chainId)
-  return config.subgraphUri
+  try {
+    const config = getOceanConfig(chainId)
+    return config.subgraphUri
```

Comment:
if we do this here as `return config?.subgraphUri` then again no need for `try/catch` as the only error here could be that `subgraphUri` can't be accessed from undefined `config`, and the `?` prevents that from erroring